dali/test/tensor_adapter.h

```diff
@@ -0,0 +1,36 @@
+#ifndef DALI_TENSOR_BATCH_ADAPTER_H
+#define DALI_TENSOR_BATCH_ADAPTER_H
+
+#include "../pipeline/data/tensor_list.h"
```

Comment:
Not sure if header paths should be relative

Reply:
Fixed